src/Widgets/DataDeleteButtonWidget/DataDeleteButtonWidgetClass.ts

```diff
@@ -4,7 +4,7 @@ export const DataDeleteButtonWidget = provideWidgetClass(
   'DataDeleteButtonWidget',
   {
     attributes: {
-      buttonStyle: ['enum', { values: ['btn-danger', 'btn-outline-primary'] }],
+      buttonColor: ['enum', { values: ['btn-danger', 'btn-outline-primary'] }],
```

Comment:
* [ ] FYI given this attribute changes more than the color `buttonStyle` was the better fit.